datastore/additional_data/sources/codelist_code.py

```diff
@@ -83,11 +98,52 @@ def update_additional_data(self, grant, additional_data):
         except (KeyError, CodelistCode.DoesNotExist):
             pass
 
+        try:
+            code = grant["locationScope"]
+            locationScope = CodelistCode.objects.get(
+                code=code, list_name="locationScope"
```

Comment:
Minor suggestion: I guess we could add constants for `CodelistCode.list_name`  https://github.com/ThreeSixtyGiving/datastore/blob/live/datastore/additional_data/models.py#L167 in the same way we have for `OrgInfoCache.org_type` https://github.com/ThreeSixtyGiving/datastore/blob/live/datastore/additional_data/models.py#L13-L31 ?

Reply:
True that would be neater. We would need to refactor to avoid there being two sources for the `list_name` string as the import code looks it up from the code list file name https://github.com/ThreeSixtyGiving/datastore/blob/live/datastore/additional_data/sources/codelist_code.py#L29

will have another look

Reply:
I think I'd prefer to leave this optimisation as it requires refactoring old code that isn't currently in scope for this small bit of work.